src/main/java/org/folio/mr/service/impl/StaffSlipContextService.java

```diff
@@ -0,0 +1,110 @@
+package org.folio.mr.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.mr.domain.dto.Instance;
+import org.folio.mr.domain.dto.InstanceContributorsInner;
+import org.folio.mr.domain.dto.Item;
+import org.folio.mr.domain.dto.MediatedRequest;
+import org.folio.mr.domain.dto.SendItemInTransitResponseStaffSlipContext;
+import org.folio.mr.domain.dto.SendItemInTransitResponseStaffSlipContextItem;
+import org.folio.mr.service.InventoryService;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class StaffSlipContextService {
+
+  private final InventoryService inventoryService;
+
+  public SendItemInTransitResponseStaffSlipContext createStaffSlipContext(MediatedRequest request) {
+    Item item = inventoryService.fetchItem(request.getItemId());
+
+    log.debug("createStaffSlipContext:: parameters item: {}", item);
```

Comment:
What if we were provided with invalid item ID? Let's make this log the first line of the method and log the request.

Reply:
done